boards/b-l072z-lrwan1/Kconfig

```
@@ -14,6 +14,7 @@ config BOARD_B_L072Z_LRWAN1
     select CPU_MODEL_STM32L072CZ
 
     # Put defined MCU peripherals here (in alphabetical order)
+    select HAS_PERIPH_ADC    
```

```suggestion
    select HAS_PERIPH_ADC
```

CI has an issue with the [trailing whitespace](https://github.com/RIOT-OS/RIOT/pull/15866/checks?check_run_id=1776409828).

You can use `git commit --amend` to add the change to the previous commit, then just `git push -f` to force push it.